spec/controllers/users/sessions_controller_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe Users::SessionsController, type: :controller do
+  describe "POST #create" do
+    before :each do
+      @request.env["devise.mapping"] = Devise.mappings[:user]
+      @user = FactoryGirl.create :user
```

Comment:
Add this   config.include FactoryGirl::Syntax::Methods to your spec_helper method inside Rspec.configure blog so you can just use the build and create methods directly